terra-jupyter-hail/Dockerfile

```diff
@@ -1,24 +1,25 @@
-FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.2
+FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.3
 USER root
 ENV PIP_USER=false
 
-COPY scripts $JUPYTER_HOME/scripts
-
-ENV SPARK_VER 2.2.3
-ENV SPARK_HOME=/usr/lib/spark
-ENV PYTHONPATH $PYTHONPATH:$SPARK_HOME/python
+ENV PYTHONPATH $PYTHONPATH:/usr/lib/spark/python
+ENV HAIL_VERSION=0.2.26
```

Comment:
Just FYI, looks like 0.2.27 is the latest

https://hail.is/docs/0.2/change_log.html

Reply:
oh yea..thanks..updated